lib/model/rwfolder.go

```diff
@@ -1689,7 +1694,7 @@ func (f *sendReceiveFolder) moveForConflict(name string) error {
 		err = nil
 	}
 	if f.MaxConflicts > -1 {
-		matches, gerr := osutil.Glob(withoutExt + ".sync-conflict-????????-??????" + ext)
+		matches, gerr := osutil.Glob(withoutExt + ".sync-conflict-????????-??????.*" + ext)
```

Comment:
Yeah according to this page it looks like it's not regex that the function accepts (which I just assumed it was) https://golang.org/pkg/path/filepath/#Glob